source/get-started/primer.txt

```diff
@@ -0,0 +1,116 @@
+.. _scala-primer:
+
+======
+Primer
+======
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: code example, design, types, preparation
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+This guide provides background about the {+driver-short+} and its
+asynchronous API before showing you how to use the driver and MongoDB in
+the 
+
+.. :ref:`Quick Start guide <>`.
+
+.. note::
+
+   See the :ref:`Installation guide <scala-install>`
+   for instructions on how to install the driver.
+
+Reactive Streams
+----------------
+
+The {+driver-short+} is built upon the `MongoDB Java Reactive Streams
+driver <https://www.mongodb.com/docs/languages/java/drivers/java-rs/current/>`__.
+The reactive stream API consists of the following components:
+
+1. ``Observable``: a custom implementation of a `Publisher <{+rs-docs+}/Publisher.html>`__
+#. ``Observer``: a custom implementation of a `Subscriber <{+rs-docs+}/Subscriber.html>`__
+#. `Subscription <{+rs-docs+}/Subscription.html>`__
+
+An ``Observable`` is a provider of a potentially unbounded number of
+sequenced elements, published according to the demand received from its
+``Observer`` or multiple instances of ``Observer``.
+
+In response to a call to ``Observable.subscribe(Observer)``, the
+possible invocation sequences for methods on the ``Observer`` class
+are given by the following protocol:
+
+.. code-block:: scala
+   :copyable: false
+
+   onSubscribe onNext* (onError | onComplete)?
+
+This means that ``onSubscribe()`` is always
+signaled, followed by a possibly unbounded number
+of ``onNext()`` signals, as requested by
+``Observer``. This is followed by an ``onError()`` signal
+if there is a failure or an ``onComplete()`` signal
+when no more elements are available, as long as
+the ``Subscription`` is not canceled.
+
+.. tip::
+
+   To learn more about reactive streams, visit
+   the `Reactive Streams documentation <http://www.reactive-streams.org>`__.
+
+Observables
+-----------
+
+The {+driver-short+} API mirrors the :driver:`Java Sync driver
+</java/sync/current/>` API and any methods that cause network 
+I/O to return a ``Observable<T>`` type, where ``T`` is the type of response
+for the operation.
+
+.. note::
+
+   All ``Observable`` types returned from the API are
+   `cold <https://projectreactor.io/docs/core/release/reference/#reactor.hotCold>`__,
```

Comment:
This link is broken (possibly temporarily based on the error message)

Reply:
I think it is a site maintenance period. I just tested the link yesterday but I'll make sure to test in the next few days to see if the site is up again!